spec/controllers/api/items_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Api::ListsController do
+  describe 'POST create' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.